src/main/groovy/nebula/plugin/dependencylock/tasks/GenerateLockTask.groovy

```diff
@@ -40,6 +40,7 @@ class GenerateLockTask extends AbstractLockTask {
     private String WRITE_CORE_LOCK_TASK_TO_RUN = "`./gradlew dependencies --write-locks`"
     private String MIGRATE_TO_CORE_LOCK_TASK_NAME = "migrateToCoreLocks"
     private static final Logger LOGGER = Logging.getLogger(GenerateLockTask)
+    private static final List<String> DEFAULT_NON_LOCKABLE_CONFIGURATIONS = ['zinc']
```

Comment:
Is this used someplace? Since it is private, an extension can't use it, for example.

Reply:
this is a carry over , will remove it soon.

Reply:
👍

Reply:
https://github.com/nebula-plugins/gradle-dependency-lock-plugin/pull/177 takes care of this